container/test.bzl

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A wrapper around container structure tests for Bazel.
+
+This rule feeds a built image and a set of config files
+to the container structure test framework."
+"""
+
+load("//container:flatten.bzl", "container_flatten")
```

Comment:
You got me excited, but this isn't used :-/

Reply:
yeah I had grandiose plans to add this as an option, but didn't get that far. we can look into supporting it later on but I'll remove for now